juntagrico/entity/subtypes.py

```diff
@@ -74,6 +74,14 @@ class SubscriptionType(JuntagricoBaseModel):
     trial_days = models.IntegerField(_('Probe-Abo Dauer in Tagen'), default=0)
     description = models.TextField(_('Beschreibung'), blank=True)
     sort_order = models.PositiveIntegerField(_('Reihenfolge'), default=0, blank=False, null=False)
+    # Interval of the subscription (e.g 2 for every 2 weeks)
+    interval = models.PositiveIntegerField(_('Intervall'), default=1, blank=False, null=False,
```

Comment:
Add `validators=[MinValueValidator(1)]` here to ensure, that interval is not 0, to avoid division by 0 later.